code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -366,9 +366,7 @@
 
 	//Put out fire
 	if(method == TOUCH)
-		M.adjust_fire_stacks(-(volume / 10))
```

Comment:
This made it so that you would need a lot of water to extinguish people - the problem is that fire extinguishers only give their puffs of water less than 1u of water each. It's either this or make fire extinguishers magically dupe water - and I'm not sure which is worse.